content/cammie.erb

```diff
@@ -104,12 +104,35 @@ socket.on('replymessage', function(obj) {
         <!-- Response Success -->
         <div id="cammieFormResponseSuccess" class="notification is-success is-light is-hidden"></div>
 
+        <!-- No username modal -->
+        <div id="cammieFormModal" class="modal">
+          <div class="modal-background"></div>
+          <div class="modal-card">
+            <header class="modal-card-head">
+              <p class="modal-card-title">Empty username message</p>
+            </header>
+            <section class="modal-card-body">
+              <p>It's pleasent for us to know who we're communicating with.</p>
```

Comment:
typo: moet pleasant zijn; maar kan mss beknopter verwoord worden als "it's nice to know who's talking to us, you can still let us know"

Reply:
Fixed